### PR TITLE
armv5te-unknown-linux-gnueabi target

### PR DESCRIPTION
**Pull request by Darneas**

### CODE REVIEW

**Comment by Darneas** [comment]
As far as I can see, the automated tests pass. How can I help to make this pull request happen?

File: README.md [hunk 1]
```diff
@@ -205,6 +205,7 @@ terminate.
 | `arm-unknown-linux-gnueabihf`        | 2.27   | 7.3.0   | ✓   | 4.1.0 |   ✓    |
 | `arm-unknown-linux-musleabi`         | 1.1.20 | 6.3.0   |     | 4.1.0 |   ✓    |
 | `arm-unknown-linux-musleabihf`       | 1.1.20 | 6.3.0   |     | 4.1.0 |   ✓    |
+| `armv5te-unknown-linux-gnueabi`      | ???    | ???     | ??? | ???   |   ?    |
```
Comment (ryankurte) [inline README.md:208]: is there a reason these are currently unspecified?

**Comment by ryankurte** [comment]
thanks for the reminder, sometimes we need a push to get things back on the stack. apart from the comment above, lgtm

File: README.md (hunk 1 above)
Comment (Darneas) [reply README.md:208]: Is there a straightforward way to print these versions in the container? I have a local one and could add these infos.

**Pull request merged**